extensions/guacamole-auth-sso/modules/guacamole-auth-sso-ssl/src/main/resources/guac-manifest.json

```diff
@@ -0,0 +1,35 @@
+{
+
+    "guacamoleVersion" : "1.5.0",
+
+    "name"      : "SSL Authentication Extension",
+    "namespace" : "ssl",
+
+    "authProviders" : [
+        "org.apache.guacamole.auth.ssl.SSLAuthenticationProvider"
+    ],
+
+    "css" : [
+        "styles/sso-providers.css"
+    ],
+
+    "js" : [ "ssl.min.js" ],
+
+    "html" : [
+        "html/sso-providers.html",
```

Comment:
What is `sso-providers.html`? I don't see this file anywhere.

Reply:
Ah - that's inherited from the "guacamole-auth-sso-base" project: https://github.com/apache/guacamole-client/blob/78c084be6568ae2ae8097daddb1cbf94903df060/extensions/guacamole-auth-sso/modules/guacamole-auth-sso-base/src/main/resources/html/sso-providers.html

Reply:
Ahhhh right